docs/src/main/asciidoc/websockets.adoc

```diff
@@ -218,6 +217,9 @@ public class ChatTest {
         @OnOpen
         public void open(Session session) {
             MESSAGES.add("CONNECT");
+            // Send a message to indicate that we are ready,
+            // as the message handler may not be registered immediately after this callback.
+            session.getAsyncRemote().sendText("_ready_");
```

Comment:
I had to add this, otherwise the test was not passing. This is the same as done in the quickstart.